.env

```diff
@@ -1 +1,4 @@
-MONGO_DEV_URI='mongodb://mariianakonechna:mary12345@127.0.0.1:27017/nosqlDB'
\ No newline at end of file
+MONGO_DEV_URI=mongodb://mariianakonechna:mary12345@127.0.0.1:27017/nosqlDB
+PORT=8000
+TOKEN_KEY=nodegmpnosqltokenkey
+NODE_ENV=test
```

Comment:
Make sure sensitive information is not hardcoded in the codebase. usually you leave the .env.example file in the code in order to highlight which variables are needed for the project, and the variables themselves are distributed through protected applications. If this is real creds for your database please delete them asap